lib/queue.rb

```diff
@@ -1,28 +1,37 @@
 class Queue
```

Comment:
This works, but it's not what was asked.  The assignment calls you to create a Queue using a circular buffer.  So... yes it's a queue and passes the tests, but it wasn't implemented as requested.